views/protocol.hbs

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <title>OIDC Doctor</title>
+    <link rel="icon" href="/images/logo-amos.ico" type="image/x-icon">
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Open Id Connect Doctor</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
 </head>
 <body>
     <main>
         <div class="container">
+            <img class="mx-auto d-block" src="/images/logo-amos.png" alt="Image"  width="430" height="300">
```

Comment:
The icon in the header is too large. This results in the rest of the content being pushed down. 
![grafik](https://user-images.githubusercontent.com/25197252/179422339-8041de29-f0cf-4a44-ad54-80b8c5aeed7a.png)

Reply:
It is resized to smaller now. Is it ok now?

Reply:
I think it is still too large. If you need some hints regarding the size, check out the initial wireframe at https://www.figma.com/file/P8WH0xN6JunPmPEUks38Qy/Untitled?node-id=0%3A1

Reply:
Our Product Owners spoke about the same size on every page in our last standup. Thanks for this link! I have reduced the size everywhere outside of the start page, because it seems to be a larger requirement than on the other pages there.

Reply:
Do you think we can shrink down the size even more or make it dynamic? If I open the application on my laptop it still takes up around 20% of my screen, which is not ideal when displaying the token-information for example.

Reply:
Do you speak about the start page or the other pages? I am surprised about the 20% of the screen on both. I wanted to keep the company name readable on the start page (therefore a little bit bigger). I will look for a dynamic adoptability in the next hour.

Reply:
I thought, that I can add the aut-resizing into the html code. I have added css styles now. It is auto-resizing now.